internal/v6provider/data_source_load_balancer_rules.go

```diff
@@ -122,12 +121,12 @@ func getLBRules(
 			return nil, err
 		}
 
-		if res.JSON200 == nil {
-			return nil, errors.New("response body is nil")
+		var totalPagesError error
+		totalPages, totalPagesError = res.JSON200.Pagination.TotalPages.Get()
+		if totalPagesError != nil {
+			return nil, totalPagesError
 		}
```

Comment:
suggestion: Instead of failing with an error here, I think it might be more appropriate to just act as if the value is 0. As there's definitely no second page to try and fetch.

Also it [looks](https://github.com/oapi-codegen/nullable/blob/v1.1.0/nullable.go#L42-L52) like the only possible errors from `.Get()` are `value is null` or `value is not specified`, which I don't believe are very useful errors to bubble upwards from here.

Something like this should do it I believe?

```suggestion
		totalPages, _ = res.JSON200.Pagination.TotalPages.Get()
```

Reply:
Ah yeah definitely :saluting_face: